cmd/fleetctl/convert_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli/v2"
+)
+
+func TestConvertFileOutput(t *testing.T) {
+	// setup the cli and the convert command
+	app := cli.NewApp()
+	app.Commands = []*cli.Command{convertCommand()}
+	app.Reader = os.Stdin
+	app.Writer = os.Stdout
+	app.Setup()
+
+	// read the expected output file
+	expected, err := ioutil.ReadFile("testdata/convert_output.yml")
```

Comment:
Please use `filepath.Join` when joining paths to maximize platform compatibility. I'll see if this can be encoded in a lint check.

Reply:
oof, yes thanks again 👍

Reply:
Heh, no big deal. Things like this are all about muscle-memory and I don't really expect anyone to remember 100% of the time.